src/aria/popups/Popup.js

```diff
@@ -731,11 +750,16 @@ module.exports = Aria.classDefinition({
                     this._containerOverflow = popupContainer.changeContainerOverflow("hidden");
                 }
 
-                var containerSize = popupContainer.getScrollSize();
+                // temporarily hides the modal mask so that it does not have an influence the size of the container
```

Comment:
typo: "on" the size

Reply:
Thank you for catching this mistake. I will fix it.